news/_posts/2020-03-12-ps17-minor-release-lifecycle.md

```diff
@@ -12,115 +12,118 @@ tags:
-After a week, if no bugs have been reported in RC1 build, we deliver the stable release. However if bugs a reported, we fix them and deliver a RC2, and so on until we build a version with no reported issues. This last Release Candidate becomes the stable release.
+- At the end of the Beta period, all regressions are fixed and maintainers deliver a Release Candidate 1 build.
+If bugs are reported, they are fixed in an RC2 build, and so on until a build has no issues reported within the following week. The last Release Candidate becomes the stable release.
 
 Expected duration: from 1 week (if RC1 is flawless) to 1 month, possibly 2 months
 
 ### Calendar
 
-If you sum all expected phases duration, you reach 6 months. This is why we expect to deliver 2 minor releases by year. 2 phases of development, 2 phases of feature freeze and 2 beta periods.
+The global duration for all the process is about 6 months. This is why we expect to deliver at most 2 minor releases per year.
+
+## Some more details
 
-# Some more details
+This lifecycle has been adopted by the PrestaShop team since 2018, following issues we had in the release process of PrestaShop 1.7.3 and 1.7.4. It seems to have proven its qualities as PrestaShop 1.7.5 and 1.7.6 releases were a lot more robust. However this comes with a cost: all of this process requires a huge investment from the whole ecosystem. Maintainers, the QA team and the Product team on the front line... but also a lot of people across the PrestaShop Company crew: marketing, partnerships, training, everybody participates in the launch, be it marketing material or communication. The community as well: submitting issues and pull requests, testing... but also upgrading their shops, modules and themes. Launching a minor version represents a huge amount work for everyone, which is why it cannot be done too often.
 
-This lifecycle has been adopted by PrestaShop team in 2018 following issues we had in the release process of PrestaShop 1.7.3 and 1.7.4. It seems to have proven its qualities as PrestaShop 1.7.5 and 1.7.6 releases were a lot more robust. However this comes with a cost: all of this process requires a huge investment from the maintainers, the QA team, the Product team ... and the community ! Also the stable release involves a lot of people of the PrestaShop crew: marketing, partnerships, training, everybody participates in the launch, be it marketing material or communication. Launching a minor version is a huge work for all the PrestaShop company and community, which is why we cannot do it too often.
+Another side note: people often ask "when will next minor version be out?"
+As you can understand from this process, there is cannot be a defined date. The answer is "Version 1.7.X will be out when a Release Candidate is delivered with no known regressions". This can take one week or three months.
```

Comment:
```suggestion
As you can understand from this process, there cannot be a defined date. The answer is "Version 1.7.X will be out when a Release Candidate is delivered with no known regressions". This can take one week or three months.
```